src/Components/StyledButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { Link } from 'expo-router';
```

Comment:
Let's put the StyledButton and its styles sheet into a folder inside `Components`!